tests/cls_test.py

```diff
@@ -50,33 +50,6 @@ def test_cls_added_when_filled(self):
         clsform.add_cls(record, fpacket, fvp_forms)
         self.assertEqual(len(fpacket), 1, "Expected packet to have CLS")
 
-    def test_partial_cls_raises_error(self):
```

Comment:
While it's possible that these tests are truly outdated, could they have been updated instead of completely removed?

Reply:
A lot of these tests were making sure the packet was not uploaded and exited with an error. Now the expected behavior would be for the packet to still upload. I can make a test for that but it seems so general. Theoretically I could make a test after every form gets added to the packet to make sure there is no error (and perhaps we should. At least with CLS it is easier because it is in its own function).

We are expecting there to be an error message printed out. I can try and look up how to test for that message, but it might take me a little time since I don't think I fully understand the way these messages are printed in NACCulator.